spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/datastore/GcpDatastoreAutoConfiguration.java

```diff
@@ -52,8 +52,7 @@
 @Configuration
 @AutoConfigureAfter(GcpContextAutoConfiguration.class)
 @ConditionalOnProperty(value = "spring.cloud.gcp.datastore.enabled", matchIfMissing = true)
-@ConditionalOnClass({ DatastoreMappingContext.class, DatastoreOperations.class,
-		ObjectToKeyFactory.class, DatastoreEntityConverter.class })
+@ConditionalOnClass({ DatastoreMappingContext.class, DatastoreOperations.class })
```

Comment:
Actually, these two are also from the same package. We only need one of them - `DatastoreOprations.class`.
We also need to make sure that we're conditional on the presence of the Datastore client library by including `com.google.cloud.datastore.Datastore`.

Reply:
added Datastore.class and removed DatastoreMappingContext.class